src/games/PlayerVsComputerGame.js

```diff
@@ -7,7 +7,30 @@ function PlayerVsComputerGame(ui, GameBoard, WinConditions) {
 PlayerVsComputerGame.prototype.initializeGame = function() {
   this.playerMarker = "X";
   this.ui.displayTurn(this.playerMarker);
-  console.log("In PvC");
+  this.ui.onSpotClicked(this.play.bind(this));
+}
+
+PlayerVsComputerGame.prototype.play = function(e) {
+  var id = this.ui.spotClicked(e, this.playerMarker);
+  var gameBoard = this.GameBoard.updateBoard(id, this.playerMarker);
+  if (this.WinConditions.endGame(gameBoard)){
+    this.ui.displayWin(this.playerMarker);
+    this.ui.disableAllClicks();
+  } else if (this.GameBoard.isBoardFull()) {
+      this.ui.displayTie();
+  } else {
+      this.playerMarker = this.GameBoard.switchMarker(this.playerMarker);
+      PlayerVsComputerGame.prototype.simpleAI(this.ui, this.GameBoard, this.playerMarker);
```

Comment:
This function could DEFINITELY use a better name! It looks like you're creating a class to me!